### ISSUE TITLE
Error on Android and Electron build on version 18.0.1

### ISSUE THREAD

**Issue by RodriSanchez1** [dpa99c/cordova-plugin-firebasex#907]
# Bug report

**CHECKLIST**
- [ x] I have read the [issue reporting guidelines](https://github.com/dpa99c/cordova-plugin-firebasex#reporting-issues)

- [ x] I confirm this is a suspected bug or issue that will affect other users

- [ ] I have reproduced the issue using the [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) or provided the necessary information to reproduce the issue.

- [x ] I have read [the documentation](https://github.com/dpa99c/cordova-plugin-firebasex) thoroughly and it does not help solve my issue.

- [x ] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**

With the new version `18.0.1` the android and electron apps are not builded.

I get the following error:
```bash
cordova-plugin-androidx-adapter: Processed 58 source files in 1044ms
Executing script found in plugin cordova-plugin-firebasex for hook "after_prepare": plugins\cordova-plugin-firebasex\scripts\after_prepare.js
The provided path "C:\Users\rodri\CBOARD\ccboard\platforms\ios" is not a Cordova iOS project.
CordovaError: The provided path "C:\Users\rodri\CBOARD\ccboard\platforms\ios" is not a Cordova iOS project.
    at new Api (C:\Users\rodri\CBOARD\ccboard\node_modules\cordova-ios\bin\templates\scripts\cordova\Api.js:98:15)
    at Utilities.getAppName (C:\Users\rodri\CBOARD\ccboard\plugins\cordova-plugin-firebasex\scripts\lib\utilities.js:71:24)
    at setupEnv (C:\Users\rodri\CBOARD\ccboard\plugins\cordova-plugin-firebasex\scripts\after_prepare.js:25:25)
    at module.exports (C:\Users\rodri\CBOARD\ccboard\plugins\cordova-plugin-firebasex\scripts\after_prepare.js:67:5)
    at runScriptViaModuleLoader (C:\Users\rodri\AppData\Roaming\nvm\v18.18.1\node_modules\cordova\node_modules\cordova-lib\src\hooks\HooksRunner.js:157:32)
    at runScript (C:\Users\rodri\AppData\Roaming\nvm\v18.18.1\node_modules\cordova\node_modules\cordova-lib\src\hooks\HooksRunner.js:136:12)
    at C:\Users\rodri\AppData\Roaming\nvm\v18.18.1\node_modules\cordova\node_modules\cordova-lib\src\hooks\HooksRunner.js:108:40
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
CordovaError: The provided path "C:\Users\rodri\CBOARD\ccboard\platforms\ios" is not a Cordova iOS project.
    at new Api (C:\Users\rodri\CBOARD\ccboard\node_modules\cordova-ios\bin\templates\scripts\cordova\Api.js:98:15)
    at Utilities.getAppName (C:\Users\rodri\CBOARD\ccboard\plugins\cordova-plugin-firebasex\scripts\lib\utilities.js:71:24)
    at setupEnv (C:\Users\rodri\CBOARD\ccboard\plugins\cordova-plugin-firebasex\scripts\after_prepare.js:25:25)
    at module.exports (C:\Users\rodri\CBOARD\ccboard\plugins\cordova-plugin-firebasex\scripts\after_prepare.js:67:5)
    at runScriptViaModuleLoader (C:\Users\rodri\AppData\Roaming\nvm\v18.18.1\node_modules\cordova\node_modules\cordova-lib\src\hooks\HooksRunner.js:157:32)
    at runScript (C:\Users\rodri\AppData\Roaming\nvm\v18.18.1\node_modules\cordova\node_modules\cordova-lib\src\hooks\HooksRunner.js:136:12)
    at C:\Users\rodri\AppData\Roaming\nvm\v18.18.1\node_modules\cordova\node_modules\cordova-lib\src\hooks\HooksRunner.js:108:40
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Within the `18.0.0` version all is working properly. I was not able to test with the Ios app because I dont have a mac.

**Expected behavior:**

Get a succesful build of the Android and Electron apps.

**Steps to reproduce:**

Just use the last 18.0.1 and `cordova android build --debug`

**Screenshots**

**Environment information**

- Cordova CLI version :
	- ` 12.0.0 (cordova-lib@12.0.1)`
- Cordova platform version
	- `Installed platforms:
  android 13.0.0
  electron
`
- Plugins & versions installed in project (including this plugin)
    - `com.lampa.startapp 6.1.6 "startApp"
cordova-clipboard 1.3.0 "Clipboard"
cordova-plugin-android-permissions 1.1.5 "Permissions"
cordova-plugin-androidx-adapter 1.1.3 "cordova-plugin-androidx-adapter"
cordova-plugin-cboard-speech-tts 0.0.1 "cboard-speech-tts"
cordova-plugin-default-browser 1.0.0 "CordovaPluginDefaultBrowser"
cordova-plugin-fbsdk 4.1.1 "Facebook Connect"
cordova-plugin-file-opener2 2.2.1 "File Opener2"
cordova-plugin-file 8.1.2 "File"
cordova-plugin-firebasex 18.0.0 "Google Firebase Plugin"
cordova-plugin-fullscreen 1.3.0 "cordova-plugin-fullscreen"
cordova-plugin-ionic-keyboard 2.2.0 "cordova-plugin-ionic-keyboard"
cordova-plugin-iosrtc 8.0.4 "iosrtc"
cordova-plugin-market 1.2.0 "Market"
cordova-plugin-media 7.0.0 "Media"
cordova-plugin-network-information 3.0.0 "Network Information"
cordova-plugin-purchase 13.11.1 "Purchase"
cordova-plugin-saf-mediastore 0.7.0 "cordova-plugin-saf-mediastore"
cordova-plugin-sign-in-with-apple 0.1.2 "cordova-plugin-sign-in-with-apple"
cordova-plugin-statusbar 2.4.3 "StatusBar"
ionic-plugin-deeplinks 1.0.22 "Ionic Deeplink Plugin"
`
- Dev machine OS and version, e.g.

    - Windows 10
        
_Android build issue:_	
- Node JS version
    - `v18.18.1`
- Gradle version
	- `8.7`
- Target Android SDK version
	- `34`

**Related code:**
```
insert any relevant code here such as plugin API calls / input parameters
```

**Console output**
<details>
<summary>console output</summary>

```

// Paste any relevant JS/native console output here

```

</details><br/><br/>

**Other information:**

I think its something within the hook after_prepare that runs this plugin. I have the p.list and the google-services.json files on my repo.

**Comment by dpa99c** [comment]
ah, looks like PR #896 broke this

**Comment by dpa99c** [comment]
Fix published in cordova-plugin-firebasex@18.0.2

**Comment by RodriSanchez1** [comment]
Nice! Thanks!
